Tools/Pipeline/Controls/PropertyGridTable.cs

```diff
@@ -176,9 +176,12 @@ private void Drawable_Paint(object sender, PaintEventArgs e)
             else
                 SetCursor(CursorType.Default);
 
+            // On windows craeting a dialog from double click will freeze
+            // the GUI thread until a click occurs so we need to call the
+            // dialog at the end of Paint event so everything gets drawn.
```

Comment:
Not sure if it's due to Eto or Windows Forms, still have to test that out. Anyway we have a fix so it's not really a priority...

Also for Gtk I have to call the dialog from the click event because Gtk drawing calls need to complete in order to send another draw call, which means if we start the dialog inside Draw event, the application will stop redrawing itself.